connector/src/yang/connector/netconf.py

```diff
@@ -335,6 +336,7 @@ def connect(self):
             'hostkey_verify': False,
             'look_for_keys': False,
             'ssh_config': None,
+            'timeout': 60
```

Comment:
Default was 30 seconds.  Is there a specific reason to increase it to 60?

Reply:
The default in ncclient is 60, and in the case that this PR is addressing would be the actual timeout